packages/build-tools/package.json

```diff
@@ -11,14 +11,16 @@
   ],
   "scripts": {
     "start": "yarn watch",
-    "watch": "tsc --watch --preserveWatchOutput",
-    "build": "tsc",
+    "watch": "yarn gql && tsc --watch --preserveWatchOutput",
+    "build": "yarn gql && tsc",
```

Comment:
nit: does it make sense to use `prewatch` & `prebuild`?

If we commit the `schema.graphql`, we could also choose not to commit `graphql-env.d.ts` and instead choose to use `gql.tada generate-output` in `build` instead